Guess game/Program.cs

```diff
@@ -10,282 +10,98 @@ namespace Guess_game
 {
     class Program
     {
+        private static void AnswerMessage(bool correct)
```

Comment:
Private methods should be placed below `Main` method (because main method is entry point and people want to see it first)